snippets/core/system-text-json/csharp/Utf8ReaderFromFile.cs

```diff
@@ -36,11 +34,11 @@ public static void Run()
                         total++;
```

Comment:
Also, it might be worth adding a comment for the `ReadAllBytes` sample, that make sure the file doesn't contain the UTF-8 BOM bytes, and if they do, slice them off before passing the text to the `Utf8JsonReader` since it expects text. Otherwise, it throws since that is considered invalid JSON.